SPECS/libcontainers-common/libcontainers-common.spec

```diff
@@ -0,0 +1,667 @@
+#
+# spec file for package libcontainers-common
+#
+# Copyright (c) 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+# commonver - version from containers/common
+%define commonver 0.14.6
+# podman - version from containers/podman
+%define podmanver 2.0.3
+# storagever - version from containers/storage
+%define storagever 1.20.2
+# imagever - version from containers/image
+%define imagever 5.5.1
+Summary:        Configuration files common to github.com/containers
+Name:           libcontainers-common
+Version:        20200727
+Release:        2%{?dist}
+License:        Apache-2.0 AND GPL-3.0+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Management
+URL:            https://github.com/containers
+#Source0:       https://github.com/containers/image/archive/v5.5.1.tar.gz
+Source0:        image-%{imagever}.tar.gz
+#Source1:       https://github.com/containers/storage/archive/v1.20.2.tar.gz
+Source1:        storage-%{storagever}.tar.gz
+Source2:        LICENSE
+Source3:        policy.json
+Source4:        storage.conf
+Source5:        mounts.conf
+Source6:        registries.conf
+#Source7:       https://github.com/containers/podman/archive/v2.0.3.tar.gz
+Source7:        podman-%{podmanver}.tar.gz
+Source8:        default.yaml
+#Source9:       https://github.com/containers/common/archive/v0.14.6.tar.gz
+Source9:        common-%{commonver}.tar.gz
+Source10:       containers.conf
+BuildRequires:  go-go-md2man
+Requires(post): grep
+Requires(post): util-linux
+Provides:       libcontainers-image
+Provides:       libcontainers-storage
+Obsoletes:      libcontainers-image
```

Comment:
Please add versions and releases to `Obsoletes` and `Provides` (note the hard-coded version for `Obsoletes`):
```
Obsoletes: libcontainers-image <= 5.5.1
Provides: libcontainers-image = %{version}-%{release}
```

Alternatively, we can drop `Obsoletes`, since we never had that package in the first place (I think, please check). We still want the version/release info for the `Provides`, though.

More detailed explanation [here](https://docs.fedoraproject.org/en-US/packaging-guidelines/#renaming-or-replacing-existing-packages).